scripts/travis-env-vars.sh

```diff
@@ -78,22 +78,22 @@ print_all_changed_files
 
 if [[ "$TEST_SUITE" == 'unit' ]]; then
   # Only run unit tests if JS files changed
-  check_for_testable_files '^karma\.conf\.js$' '^packages/.+\.js$' '^test/unit/.+\.js$'
+  check_for_testable_files '^karma\.conf\.js$' '^packages/.+\.(js|ts)$' '^test/unit/.+\.js$'
 fi
 
 if [[ "$TEST_SUITE" == 'lint' ]]; then
   # Only run linter if JS/Sass files changed
-  check_for_testable_files '\.(js|css|scss)$'
+  check_for_testable_files '\.(js|ts|css|scss)$'
 fi
 
 if [[ "$TEST_SUITE" == 'build' ]]; then
   # Only run build if package JS/Sass files changed
-  check_for_testable_files '^packages/.+\.(js|css|scss)$'
+  check_for_testable_files '^packages/.+\.(js|ts|css|scss)$'
 fi
 
-if [[ "$TEST_SUITE" == 'closure' ]]; then
-  # Only run closure test if package JS files changed
-  check_for_testable_files '^packages/.+\.js$'
+if [[ "$TEST_SUITE" == 'typescript' ]]; then
```

Comment:
Should this branch be removed? (See also comment RE removing the task in .travis.yml)